lib/email/receiver.rb

```diff
@@ -238,16 +238,20 @@ def is_bounce?
 
     def handle_bounce
       @incoming_email.update_columns(is_bounce: true)
+      mail_error_statuses = Array.wrap(@mail.error_status)
```

Comment:
This just results in `[]` if `@mail.error_status` is NULL. I am not sure if the status can have more than one value, so just went with this to be on the safe side

Reply:
From https://www.iana.org/assignments/smtp-enhanced-status-codes/smtp-enhanced-status-codes.xhtml it feels like it may just be one? W.XXX.YYY

Reply:
Yeah I would assume so, but there was an `array.wrap` there before, so don't want to change things up too much :)